plugins/scaffolder-backend/src/scaffolder/dryrun/createDryRunner.ts

```diff
@@ -94,18 +91,23 @@ export function createDryRunner(options: TemplateTesterCreateOptions) {
       ]),
     });
 
-    const dryRunId = uuid();
+    // Extracting contentsPath and dryRunId from the baseUrl
+    const baseUrl = input.templateInfo.baseUrl;
+    if (!baseUrl) {
+      throw new Error('baseUrl is required');
+    }
+    const basePath = fileURLToPath(new URL(baseUrl));
+    const contentsPath = basePath.replace('template.yaml', '');
```

Comment:
you can use `fs.dirname` here instead of parsing like this.

Reply:
I wasn't able to find a method in `fs` for `fs.dirname`. I used `path.dirname` hoping that's what you meant.